service/controller/app/v1/resource/appnamespace/create.go

```diff
@@ -0,0 +1,7 @@
+package appnamespace
+
+import "context"
+
+func (*Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
```

Comment:
Let's have a comment explaining why we don't need to implement `EnsureCreated`.

Reply:
I will put the comment inside the function.

Reply:
But will it be godoc? Is this for the internal function?

I'd like to have it here too. As when I'm trying to figure out what a resource does this is where I look first.

Reply:
I will put a godoc as ```// EnsureCreated does not need to fetch the status of namespace since operator only
// need to take consider of deletion event.``` WDYT?

Reply:
Thanks and can we go with this?

> EnsureCreated is not implemented because we only need to consider the status of the namespace on deletion events.